datasets/modeled/buildings/dataset.json5

```diff
@@ -0,0 +1,116 @@
+{
+  dataset_id: "decarb_2023_buildings",
+  dataset_type: "modeled",
+  sector_description: "",
+  description: "",
+  origin_creator: "Aven Satre Meloy",
+  origin_organization: "",
+  origin_contributors: [
+  ],
+  origin_project: "",
+  origin_date: "",
+  origin_version: "",
+  data_source: "",
+  source: "",
+  data_classification: "low", // TODO: confirm
+  tags: [
+  ],
+  trivial_dimensions: [
+    "scenario",
+    "subsector",
+    "weather_year",
+  ],
+  data_schema: {
+    table_format: {
+      format_type: "unpivoted",
+      value_column: "value",
+    },
+    data_schema_type: "one_table",
+  },
+
+  dimensions: [
+    {
+      type: "geography",
+      name: "Region",
+      display_name: "Region",
+      file: "dimensions/regions.csv",
+      module: "dsgrid.dimension.standard",
+      class: "GeographyDimensionBaseModel",
+      description: "",
```

Comment:
Would be good to provide the URL to a general description and/or map.

Reply:
Here's the [link](https://www.eia.gov/outlooks/aeo/pdf/nerc_map.pdf) that we direct to typically.